src/main/java/mate/jdbc/service/DriverServiceImpl.java

```
@@ -0,0 +1,38 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Driver;
+
+@Service
+public class DriverServiceImpl implements DriverService {
+    @Inject
+    private DriverDao driverDao;
+
+    @Override
+    public Driver create(Driver driver) {
+        return driverDao.create(driver);
+    }
+
+    @Override
+    public Driver get(Long id) {
+        return driverDao.get(id).get();
```

Consider handling the case when `driverDao.get(id)` returns an empty Optional. You can use `orElseThrow` to throw a custom exception if the driver is not found. 🕵️